### ISSUE TITLE
Rust docset not completely rendered ("Loading content...")

### ISSUE THREAD

**Issue by cyphar** [zealdocs/zeal#1195]
If you try to load the documentation for [`std::iter::Iterator`](https://doc.rust-lang.org/std/iter/trait.Iterator.html), the documentation has been stripped of its JavaScript and the method documentation is missing entirely (all that is there is a `Loading content...` placeholder). I think this is because Rust's documentation uses JavaScript to fill in the documentation section for some trait and struct methods.

<details>
<summary>Zeal Screenshot -- Iterator</summary>

![2020-02-26-203312_1426x339_scrot](https://user-images.githubusercontent.com/2888411/75331710-4bbb4b00-58d7-11ea-9777-4ca44873b4e3.png)

</details>

<details>
<summary>docs.rust-lang.org Screenshot -- Iterator</summary>

![2020-02-26-203332_990x453_scrot](https://user-images.githubusercontent.com/2888411/75331699-48c05a80-58d7-11ea-9767-e96f15c37008.png)
![2020-02-26-203257_1026x563_scrot](https://user-images.githubusercontent.com/2888411/75331713-4c53e180-58d7-11ea-9655-64a771ba4cc5.png)

</details>

This happens for more than just `Iterator`, the same issue is present for [`std::option::Option`](https://doc.rust-lang.org/std/option/enum.Option.html):

<details>
<summary>Zeal Screenshot -- Option</summary>

![2020-02-26-203858_572x759_scrot](https://user-images.githubusercontent.com/2888411/75332161-0c412e80-58d8-11ea-95e1-50c69f02b091.png)

</details>

<details>
<summary>docs.rust-lang.org Screenshot -- Option</summary>

![2020-02-26-203845_993x669_scrot](https://user-images.githubusercontent.com/2888411/75332167-0d725b80-58d8-11ea-8152-1607c15b1f1f.png)

</details>

... and any other Rust stdlib types I've tried to look up (such as `std::vec::Vec`). I'm not sure if this is a problem with the Rust docset itself or if it's how Zeal renders it, but it renders the offline documentation practically useless because you can't get any documentation about methods on stdlib types and traits.

**Comment by condy0919** [comment]
Same issue after updating docsets

**Comment by trollixx** [comment]
I don't think it's an issue with Zeal though, same problem occurs in Chrome and Firefox when loading the page. No errors in the console.

**Comment by cyphar** [comment]
@trollixx Are you running with a JavaScript blocker? The screenshots I posted in the original bug report were rendered with Firefox on Linux and https://docs.rust-lang.org/ works fine on any device or browser I've tried.

**Comment by trollixx** [comment]
I was talking about pages from the docset. You can open them in an external browser, there's an option in the context menu.

**Comment by cyphar** [comment]
Ah okay. However I did just try that, and it does render the methods of `std::option::Option` (though they are rendered quite differently to https://docs.rust-lang.org/.

![from context menu](https://user-images.githubusercontent.com/2888411/76771644-c2bb7380-67f3-11ea-8d1d-492c1be7cd60.png)

**Comment by ruffson** [comment]
Not sure if this is related, but I also noticed that some entries (or many actually) are incomplete. I just updated the Rust docs again and checked e.g. `std::collections::HashMap`.

Here is what zeal shows me:
![zeal_hashmap](https://user-images.githubusercontent.com/1185812/79685877-4544b200-823c-11ea-8577-4497a2feeaac.png)


And here is what the [official docs](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.entry) show (I had to collapse the methods to show more than the first few. Although still could not fit all.):

![firefox_hashmap](https://user-images.githubusercontent.com/1185812/79685878-4e358380-823c-11ea-884e-2eca739520bd.png)


Sadly this makes it quite useless for Rust right now.
